### ISSUE TITLE
508-defect-1 ❗ Launchblocker [SCREENREADER] Missing and unlinked numbers

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-cms#11488]
# [508-defect-1 :exclamation: Launchblocker]

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1

Issue Title: Defect level 1! missing and unlinked numbers
Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As a user, I expect that items such as phone numbers which are linked throughout a site will be linked consistantly. I expect that when asked to call a number, particularly in a crisis situation, that number will be provided and linked so I can easilly call using my smart phone.

<h2>Details</h2>
On the Health Services page, the following numbers are either missing or not linked:
<ul>
<li>Under Pharmacy, phone numbers aren’t linked.</li>
<li>Under advice nurse, the user is advised to contact an advice nurse but there is no phone number listed.</li>
<li>Under suicide prevention/veteran’s crisis line, the phone number for the crisis line isn’t linked.</li>
</ul>

Acceptance Criteria
- [ ] The phone number for the crisis line under "suicide prevention" is linked
- [ ] The phone number is provided and linked under "advice nurse"
- [ ] All phone numbers are linked under "pharmacy"

Environment

Steps to Recreate

Proposed Solution (if known)

WCAG or Vendor Guidance (optional)

Screenshots or Trace Logs

**Comment by laflannery** [comment]
If you just add the HTML I _think_ this should work:
`<p><a aria-label="9 8 8." href="tel:988">988</a>, Select 1<br>
Hours:&nbsp;24/7</p>`

However:

1. I do not know for sure
2. Now that I know this is the main service taxonomy content, is there an approval process that we need to confirm we can change this? Who is in charge of this content?

**Comment by dsinla** [comment]
@laflannery  My previous comment wasn't a specific question for you, as much as it was notes in the ticket for whomever was about to tackle it 😄 
I think @omahane is already digging in and finding the where the root of this issue exists:
https://dsva.slack.com/archives/C0FQSS30V/p1667842276693919

**Comment by laflannery** [comment]
@dsinla Got it, thanks!

Also, FYI I added back the `508/accessibility` and `508-defect-1` labels, I use these for tracking and for finding my issues

**Comment by omahane** [comment]
Checking https://www.va.gov/greater-los-angeles-health-care/locations/west-los-angeles-va-medical-center/
This is the page in prod where the description is set:
https://prod.cms.va.gov/taxonomy/term/35/edit?destination=/admin/structure/taxonomy/manage/health_care_service_taxonomy/overview

**Comment by omahane** [comment]
@dsinla I'll only be able to verify that that the HTML is rendering on the node:view screen, not on the front end where the React is running. That's a @ryguyk or @maxx1128 question. I'm testing on my local to see if the HTML will show to the editor as rendered or plain text.

But @laflannery 's question about approval is still pertinent. I think @davidconlon manages the VA Services taxonomy. Correct?

**Comment by davidconlon** [comment]
There is a filter on the front end for linking phone numbers that needs to be applied to this field I believe.
The health service taxonomy is plain text.

**Comment by omahane** [comment]
I made these changes to the demo environment and ran content release. More details as they are made available.
![main-ersyymdhh8rrzxlevrh1xsmeyx7pzcpv demo cms va gov_taxonomy_term_35_edit_destination=_admin_structure_taxonomy_manage_health_care_service_taxonomy_overview](https://user-images.githubusercontent.com/766573/200644481-fbc4b7d4-457a-4513-b663-773f4c32c53e.png)

**Comment by omahane** [comment]
VAMC Page on front end:
<img width="1114" alt="Screen Shot 2022-11-08 at 12 49 49 PM" src="https://user-images.githubusercontent.com/766573/200650534-de333a57-97cb-4760-9499-2745a5acb929.png">

**Comment by omahane** [comment]
@laflannery The term descriptions in VAMC and Vet Center for Suicide Prevention have been updated on prod:
![Screen Shot 2022-11-08 at 4 16 48 PM](https://user-images.githubusercontent.com/766573/200687876-f5303ff5-80dc-481f-a296-0a173f4ef395.png)

**Comment by omahane** [comment]
The terms "Advice Nurse" and "Pharmacy" do not use the 988 number, so those entries must be specific to individual facility services. We'll need an audit of those to find all uses.

**Comment by laflannery** [comment]
@omahane When I first reviewed this I confirmed that the other 2 numbers had been updated, only the Crisis number still needed updating (look at my comment from Sep 29).  I also just confirmed that the numbers in Advice Nurse and Pharmacy are not taxonomy content but service descriptions, [here's an example](https://prod.cms.va.gov/puget-sound-health-care/health-services/advice-nurse-at-va-puget-sound-health-care).  

My point being that this instance was a report of 1 particular VAMC Health Services page. Now that we have resolved the Crisis Line ([which I can see here](https://www.va.gov/puget-sound-health-care/health-services/)), this ticket should be resolved. Let me know if this make sense

**Comment by omahane** [comment]
@laflannery OK, thanks for the confirmation. Do we need another ticket for updating the service descriptions? Also, do we have a report of all of those? They look like they'll be hard to find.
